replica-common/src/main/java/com/graphhopper/CustomGraphHopperGtfs.java

```diff
@@ -72,29 +66,6 @@ protected void registerCustomEncodedValues(EncodingManager.Builder emBuilder) {
         StableIdEncodedValues.createAndAddEncodedValues(emBuilder);
     }
 
-    /**
-     * Override creation of OSM reader to read the file once at initialization time, for the sole purpose of storing
-     * OSM information that will be used later in the export script.
-     *
-     * Note that this approach requires reading the OSM file twice: once during the static initialization code, and
-     * once during the call to importOrLoad() in ExportCommand.java, which is where the modified storeOsmWayID method
-     * overridden below is called to populate the ghIdToOsmId map.
-     *
-     * todo: figure out if it's possible to eliminate the need for two OSM read operations
-     */
-    @Override
-    protected DataReader createReader(GraphHopperStorage ghStorage) {
-        OSMReader reader = new OSMReader(ghStorage) {
-            // Hacky override used to populate GH ID -> OSM ID map; called during standard GH import process
-            @Override
-            protected void storeOsmWayID(int edgeId, long osmWayId) {
-                super.storeOsmWayID(edgeId, osmWayId);
-                ghIdToOsmId.put(edgeId, osmWayId);
-            }
-        };
-        return initDataReader(reader);
-    }
-
     public void collectOsmInfo() {
```

Comment:
This class, `CustomGraphHopperGtfs`, is only used in the `import` command, where we're no longer parsing/storing OSM data (that happens in a separate nationwide export step). We need to keep this `collectOsmInfo()` function just for the purpose of unit tests